### PR TITLE
Fix/bridge service

### LINKED ISSUE #150: Added storage api for the bridge rest api

### PR DESCRIPTION
**Pull request by cool-develope**
Closes #150

### What does this PR do?

- Added the storage apis for the bridge rest service
- Make the workflow config to skip e2e test

### Reviewers

Main reviewers:

- @ARR552 
- @arnaubennassar

### CODE REVIEW

File: db/pgstorage/pgstorage.go [hunk 1]
```diff
@@ -268,17 +268,31 @@ func (p *PostgresStorage) ConsolidateBatch(ctx context.Context, batch *etherman.
...
+	const getDepositSQL = "SELECT orig_net, token_addr, amount, dest_net, dest_addr, deposit_cnt, block_id, network_id, tx_hash, metadata FROM syncv2.deposit WHERE network_id = $1 AND deposit_cnt = $2"
+	err := p.getExecQuerier(dbTx).QueryRow(ctx, getDepositSQL, networkID, depositCounterUser).Scan(&deposit.OriginalNetwork, &deposit.TokenAddress, &amount, &deposit.DestinationNetwork, &deposit.DestinationAddress, &deposit.DepositCount, &deposit.BlockID, &deposit.NetworkID, &deposit.TxHash, &deposit.Metadata)
+	deposit.Amount, _ = new(big.Int).SetString(amount, 10) //nolint:gomnd
+	return &deposit, err
 }
 
 // GetLatestExitRoot gets the latest global exit root.
 func (p *PostgresStorage) GetLatestExitRoot(ctx context.Context, dbTx pgx.Tx) (*etherman.GlobalExitRoot, error) {
-	return nil, nil
+	return p.GetLatestL1SyncedExitRoot(ctx, dbTx)
```
Comment (ARR552) [inline db/pgstorage/pgstorage.go:295]: why don't we return the grpc ger instead of the l1 ger?

File: db/storage_test.go [hunk 2]
```diff
@@ -29,100 +23,6 @@ func TestExitRootStore(t *testing.T) {
...
-		OriginalNetwork:    1,
-		Token:              common.HexToAddress("0x29e885edaf8e4b51e1d2e05f9da28161d2fb4f6b1d53827d9b80a23cf2d7d9fc"),
-		Amount:             amount,
-		DestinationAddress: common.HexToAddress("0x29e885edaf8e4b51e1d2e05f9da28161d2fb4f6b1d53827d9b80a23cf2d7d9fc"),
-		BlockNumber:        1,
-		BlockID:            id,
-		NetworkID:          2,
-		TxHash:             common.HexToHash("0xc67996b03ac2ca401822b5be568e828f432121dd07ddedb306d9203e67675db8"),
-	}
-	err = storage.AddClaim(ctx, &claim, nil)
```
Comment (ARR552) [inline db/storage_test.go:72]: What about all these tests??

File: db/pgstorage/pgstorage.go (hunk 1 above)
Comment (cool-develope) [reply db/pgstorage/pgstorage.go:295]: it's not completed yet because of missing trusted globalexitroot

File: db/storage_test.go (hunk 2 above)
Comment (cool-develope) [reply db/storage_test.go:72]: https://github.com/0xPolygonHermez/zkevm-bridge-service/issues/143
https://github.com/0xPolygonHermez/zkevm-bridge-service/issues/146

**Review by fgimenez — APPROVED** [review]

File: db/pgstorage/pgstorage.go (hunk 1 above)
Comment (ARR552) [reply db/pgstorage/pgstorage.go:295]: ok, just take it on mind

**Pull request merged**